jib-cli/src/main/java/com/google/cloud/tools/jib/cli/ArtifactProcessor.java

```diff
@@ -14,15 +14,15 @@
  * the License.
  */
 
-package com.google.cloud.tools.jib.cli.jar;
+package com.google.cloud.tools.jib.cli;
 
 import com.google.cloud.tools.jib.api.buildplan.FileEntriesLayer;
 import com.google.common.collect.ImmutableList;
 import java.io.IOException;
 import java.util.List;
 
-/** Interface to create layers and compute entrypoint from jar file contents. */
-public interface JarProcessor {
+/** Interface to create layers and compute entrypoint from JAR or WAR file contents. */
```

Comment:
Update the Javadocs of the methods too. They say only "jar."

Reply:
Good catch.